### ISSUE TITLE
Warning for 'CLEAR DATA' & Show all data related

### ISSUE THREAD

**Issue by rockinfreakshow** [navjagpal/browser-timetracker#30]
A warning popup when trying to clear data so as to not delete accidentally since it happened quite a few times:(

Fit 'Show All data' in the style of rows columns so as to fit more data into one window for easy screenshot purpose rather than scrollable type bcoz it takes multiple screenshots to cover entire day stats for my project_team 

Or this is a bit far but is there a way to include 'Take screenshot' option for the entire page till bottom. The default full page screenshot addons on chrome only work on webpages and not chrome:extension//

**Comment by navjagpal** [comment]
For the screenshot stuff, have you considered downloading the raw CSV file instead? That would make it easier to play with the data. You can get the raw CSV by clicking "Options" then "Download CSV"

I can add a warning popup for the clearing of data.

**Comment by rockinfreakshow** [comment]
Hey.. thnx a lot for the warning popup thing.. It would really really help:)

hmm.. the team is around 25 and we ask them to link screenshot to the time_data as a proof that they are not faking the time span spent on tool;) ... no probs... as of now they are submitting 2/3 screenshots on whole to cover the full data.. we'll stick with that!

**Comment by navjagpal** [comment]
You have a team of 25 using this tool to keep track of time? That's cool. If there are things I can do to make it easier for your team, and aren't too expensive to implement, I'd be open to helping out.

**Comment by rockinfreakshow** [comment]
yes we tried lot of trackers and fixated on urs actually & the main reason being there is no way for the employee to alter the idle time from 60sec whereas others gave the option to alter it which can be misused.
The second reason being its available in mins directly & that saves us a lot of time from the calculations if its in hrs-mins-sec style.

wow.. that would be really grateful bcoz honestly I really need quite a few modifications if possible.  I'll get back to you with a proper list of things

**Comment by navjagpal** [comment]
I pushed a change to show a confirmation dialog before clearing stats. It should be live within an hour or so.

**Comment by rockinfreakshow** [comment]
wow.. its working.. one thing solved.. thnx a ton:)

I've jolted down quite a few modification requests below & I am not sure how feasible they are since I have limited knowledge of coding . Attached a screenshot below for a easy way to put it

**REALLY-REALLY-ESSENTIAL**
1.  Below Last Reset, I would like to see any of the following conditions whichever will really work out in easy way
- [ ] Last Time when 'Allow in Incognito' is toggled for this addon in current session [OR]
- [ ] jus a Boolean function to tell if 'Allow in Incognito' setting is messed with (TRUE/FALSE) [OR]
- [ ] Increment Counter to tell how many times 'chrome://extensions' is visited [OR]
- [ ] Once again jus a boolean function to test if 'chrome://extensions' is available in the list of 100s of sites employee might have visited (TRUE/FALSE)

2. Below Last Reset n Last Incog data, I wud like to see the count of Managed Sites (IF ANY ADDED BY THE EMPLOYEE TO MASK DATA ELSE ITS ZERO or INVISIBLE jus like SHOW ALL BUTTON which only pops-up when the count of websites crosses a certain threshold)

3. Give us an option to check/enable such as 'IF CLEARING DATA - AUTO DOWNLOAD CSV FILE' bcoz if someone cleared the data w/out taking a screenshot there is no backup for us to go back to:/

![screenshot](https://cloud.githubusercontent.com/assets/23070284/19834575/0a5895a0-9e8b-11e6-97e2-d6513599ed3c.png)

**NOT-SO-PRIORITY**
1. Take screenshot feature as in this addon
![screenshot 1](https://cloud.githubusercontent.com/assets/23070284/19834692/81e11e4e-9e90-11e6-9028-c0e49efd8c59.png)
